src/Log/Log.php

```diff
@@ -369,7 +369,9 @@ public static function write($level, $message, $context = [])
             }
 
             $correctLevel = empty($levels) || in_array($level, $levels);
-            $inScope = empty($scopes) || array_intersect($context['scope'], $scopes);
+            $inScope = $scopes === [] || $scopes === null ||
+                $scopes === false && empty($context['scope']) ||
```

Comment:
this is difficult to read... I'd ask the same as markstory... ins't it the same as just saying `empty($scope)` ?